devel/openssl3/Portfile

```diff
@@ -53,26 +53,23 @@ checksums           rmd160  f7d3736a023dcf7d40db016182ca6d1de5a6fa69 \
                     sha256  777cd596284c883375a2a7a11bf5d2786fc5413255efab20c50d6ffe6d020b7e \
                     size    18055752
 
-# 3.2.0 is currently broken for OS < 10.14, see #68766
-if {${os.platform} eq "darwin" && ${os.major} < 18} {
-
-    subport ${name}-devel {
-        conflicts           ${name}
-    }
```

Comment:
Strictly speaking this silently drops the `openssl-devel` port, but I'm fine with that.

Reply:
Ah, you're right.  I guess strictly speaking, it should have been obsoleted in favor of the main port.  But probably the only users were a few people (besides myself) testing the new version on the older platforms, so it's not clear it's worth worrying about.

Reply:
Actually, properly obsoleting this is more important than appears at first glance, due to the deficient handling of existing installs that don't match current Portfiles.  Although each installed instance keeps a copy of the Portfile used to create it, base does a poor job of using that information, leading to at least two known problems:

1. Activating a downlevel port instance apparently computes its dependencies based on its _current_ Portfile, not the saved one.
2. Attempts to clean an installed port with no current Portfile are refused, based on the lack of a current PortIndex entry.

Thus, I think it's worth properly obsoleting the `opensl3-devel` port.  PR coming.